DatadogRUM/Tests/Mocks/RUMDataModelMocks.swift

```diff
@@ -0,0 +1,470 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import TestUtilities
+
+@testable import DatadogRUM
+
+extension RUMUser {
+    static func mockRandom() -> RUMUser {
```

Comment:
nit/non blocking 
prefer `makeRandomMock`

Reply:
We are following a pattern for creating mocks: [AnyMockable ad RandomMockable](https://github.com/DataDog/dd-sdk-ios/blob/feature/v2/TestUtilities/Mocks/FoundationMocks.swift#L36L42)
Not sure why this one does not comply to the protocol 🤔